runner_scripts/cura_installer_filename.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+import os
+
+
+def set_installer_filename(args):
+    os_name = {"Linux": "linux", "Windows": "win64", "macOS": "macos"}.get(args.os)
+    enterprise = "-Enterprise" if args.enterprise == "true" else ""
+    internal = "-Internal" if args.internal == "true" else ""
+
+    installer_filename = f"UltiMaker-Cura-{os.getenv('CURA_VERSION_FULL')}{enterprise}{internal}-{os_name}-{args.architecture}"
```

Comment:
I like this pattern a bit better. 

The following line
```
enterprise = "-Enterprise" if args.enterprise == "true" else ""
```
relies on the fact that it is _not_ the first name in the string-pattern. In the other pattern you are free to hustle any of the strings around and it would still produce a sensible output.
```suggestion
    os_name = {"Linux": "linux", "Windows": "win64", "macOS": "macos"}.get(args.os)
    enterprise = "-Enterprise" if args.enterprise == "true" else ""
    internal = "-Internal" if args.internal == "true" else ""
    
    installer_filename_args = ["UltiMaker-Cura", os.getenv('CURA_VERSION_FULL')]
    if args.enterprise == "true":
        installer_filename_args.append("Enterprise")
    if args.internal == "true":
        installer_filename_args.append("Internal")
    installer_filename_args.append(os_name)
    installer_filename_args.append(args.architecture0
    installer_filename = "-".join(installer_filename_args)
```